oracle/Dockerfile

```diff
@@ -18,6 +18,9 @@ ENV USER_ID ${USER_ID:-1001}
 ARG GROUP_ID 
 ENV GROUP_ID ${GROUP_ID:-1001}
 
+# Install git because it's very useful to have that should the version be extracted from git
```

Comment:
Would be better have this in a different pr, and applied to all images

Reply:
I would have to check all the listed base images in the workflows for git installations